src/tests/api/BavHappyPath.test.ts

```diff
@@ -56,9 +58,31 @@ describe("Test BAV End Points", () => {
         const authCode = authResponse.data.authorizationCode;
         const authRepeatResponse = await authorizationGet(origSessionId);
         const authRepeatResponseCode = authRepeatResponse.data.authorizationCode;
-        expect(authCode).not.toEqual(authRepeatResponseCode); 
+        expect(authCode).not.toEqual(authRepeatResponseCode);
+    });
+
+    it.skip("BAV CRI /userinfo Happy Path", async () => {
+        // const documentSelectionResponse = await postDocumentSelection(docSelectionData, sessionId);
```

Comment:
Can you remove this please? Looks like F2F logic - This will eventually be replaced with /verifyAccount